src/components/Calculator.js

```diff
@@ -1,43 +1,44 @@
 import React from 'react';
+import { calculate, type, clear } from './logic/calculate';
 
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] I noticed that you disable a lot **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: